### PR TITLE
feat(glossaire): add glossary page

### PR DESCRIPTION
**Pull request by lionelB**
This PR add glossary page and definition 
need to add the snippet on content

related to #1186

### CODE REVIEW

File: packages/code-du-travail-data/dataset/glossaire/index.js [hunk 1]
```diff
@@ -1,36 +1,22 @@
...
+  const response = await fetch(RECORDS_URL);
+  const items = await response.json();
+  return items.data.map(({ title, abbrs, variants, definition, refs }) => ({
+    title,
+    slug: slugify(title),
+    abbrs,
+    variants,
+    definition,
+    refs
+  }));
```
Comment (revolunet) [inline packages/code-du-travail-data/dataset/glossaire/index.js]: `abbrs` and `variants` are multiline string, we should split on `\n` on fetch ?

**Comment by revolunet** [comment]
shouldnt we put that in `datafiller/glossaire.data.json` (we have themes and prequalified there and maybe more later)

File: packages/code-du-travail-data/dataset/glossaire/index.js (hunk 1 above)
Comment (lionelB) [reply packages/code-du-travail-data/dataset/glossaire/index.js]: Alright!  we should translate to an array before json

**Review by revolunet — APPROVED** [review]

**Comment by lionelB** [comment]
j'attend les retours de @virginielastisneres  et on merge, on fera les tooltip dans une deuxieme PR

**Comment by lionelB** [comment]
@revolunet bon finalement les tooltip sont aussi là 😅

**Pull request merged**